lib/internal/source_map/prepare_stack_trace.js

```diff
@@ -61,10 +61,15 @@ const prepareStackTrace = (globalThis, error, trace) => {
     try {
       // A stack trace will often have several call sites in a row within the
       // same file, cache the source map and file content accordingly:
-      const fileName = t.getFileName();
+      let fileName = t.getFileName();
+      let generated = false;
+      if (fileName === undefined) {
+        fileName = t.getEvalOrigin();
```

Comment:
Does `getEvalOrigin` return the origin filename and line #?

Reply:
It returns the filename parsed from the magic comment in the eval source text: https://source.chromium.org/chromium/chromium/src/+/main:v8/src/objects/call-site-info.cc;l=218?q=call-site-info.cc

Reply:
Gotcha, so a format that looks something like this:

`eval at <anonymous> (REPL1:1:1), <anonymous>:1:1`